core/util/makeConfig.js

```diff
@@ -1,32 +1,44 @@
 const path = require('path');
+const fs = require('fs');
 const extendConfig = require('./extendConfig');
+const logger = require('../util/logger')('init');
 
 const NON_CONFIG_COMMANDS = ['init', 'version', 'stop'];
 
-function projectPath (config) {
+function projectPath () {
   return process.cwd();
 }
 
 function loadProjectConfig (command, options, config) {
+  options = options || {}; // make sure options is an object
+
   // TEST REPORT FILE NAME
-  const customTestReportFileName = options && (options.testReportFileName || null);
-  if (customTestReportFileName) {
-    config.testReportFileName = options.testReportFileName || null;
+  if (options.testReportFileName) {
+    config.testReportFileName = options.testReportFileName;
   }
 
-  let customConfigPath = options && (options.backstopConfigFilePath || options.configPath);
-  if (options && typeof options.config === 'string' && !customConfigPath) {
-    customConfigPath = options.config;
+  // list of "config" files to search for
+  const configFiles = ['backstop.json', 'backstop.js'];
```

Comment:
```suggestion
  const configFiles = ['backstop.json', 'backstop.config.json', 'backstop.js', 'backstop.config.js'];
```

Reply:
@maxfenton Do you REALLY need the support `backstop.config.*`? 

I personally don't mind if we add them, still I don't see it necessary. 

@garris  ? What do you say?

Reply:
@klodoma I have a coworker who — today — said he prefers it on his projects because it’s more clear when looking at the root of a project what's a config file. I'd feel bad if I didn't at least make this suggestion for his sake and since it’s probably the default some other users have been using with the config flag.

Reply:
And it’s the default name in the documentation here: https://github.com/garris/BackstopJS?tab=readme-ov-file#working-with-your-config-file